app/views/reviews/new.html.erb

```diff
@@ -1,9 +1,3 @@
-<% flash.each do |type, msg| %>
-  <div>
-    <font size="5" color="red"><%= msg %></font>
-  </div>
-<% end %>
-
```

Comment:
Removed this flash message as it was already accounted for in application.html and was redundant